locales/en/bonus/bpd/optInPaymentMethods.md

```diff
@@ -23,7 +23,7 @@ For complaints and information on refunds, go to the website of <a href="https:/
   <li>
     <body>
       <b>If you keep them</b>, you’ll be able to use them to take part in discounts, bonuses and future initiatives or for other features, such as in-app payments. It’s not a final decision: you can delete them at any time.
-      The data controller will no longer be the Ministry of Economy and Finance, but App IO. See the Privacy Policy for more information. 
+      The data controller will no longer be the Ministry of Economy and Finance, but PagoPA S.p.A., the company in charge of managing the IO app. See the [Privacy Policy](https://io.italia.it/app-content/tos_privacy.html) for more information. 
```

Comment:
Scanning around this file, I see that [we use <a> elements](https://github.com/pagopa/io-app/pull/3777/files#diff-ac3d1e31b53b07b2e19fbbda58f0e742f837cad5911ca098ee42a09b3220499aR12) rather than Markdown link notation.
I might be a tad too cautious here, but I think the former is the way to go.

Reply:
@pietro909 is right, we should use the html version in this case.